Tools/autotest/arducopter.py

```diff
@@ -1671,33 +1671,121 @@ def fly_optical_flow_limits(self):
 
     # fly_autotune - autotune the virtual vehicle
     def fly_autotune(self):
+        """Test autotune mode"""
 
-        self.takeoff(10)
+        ex = None
+        self.context_push()
```

Comment:
Why the extra context here?

There's an implicit context further up.

Typically the only reason to have your own context is because you want to run multiple phases in the one test (and want a convenient way to reset the parameters), or you've poked some reboot-required parameter and had to reboot the vehicle for stuff to happen.  The contexts don't currently handle reboot-required parameters, so you have to do that bit yourself.

Reply:
Removed